itests/src/test/java/org/apache/unomi/itests/BasicIT.java

```diff
@@ -93,6 +97,27 @@ public class BasicIT extends BaseIT {
     @Inject @Filter(timeout = 600000)
     protected DefinitionsService definitionsService;
 
+    @Inject @Filter(timeout = 600000)
+    protected ScopeService scopeService;
+
+    @Before
+    public void setUp() throws InterruptedException {
+        Scope scope = new Scope();
+        scope.setItemId(TEST_SCOPE);
+        Metadata metadata = new Metadata();
+        metadata.setName("Test scope");
+        metadata.setId(TEST_SCOPE);
+        scope.setMetadata(metadata);
```

Comment:
Maybe we could refactor this creation to TestUtils.newScope(String scopeId) as we use it multiple times in the tests ?